### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by g0dm0d3** [coverlet-coverage/coverlet#1273] [General]
Hi! When do you plan to release? We are really looking forward to next release.
I express my gratitude for the work done and for the great product in general.
Merry Christmas and Happy new year! =)

**Comment by daveMueller** [comment]
There was a new version release yesterday [3.1.1](https://github.com/coverlet-coverage/coverlet/releases/tag/v5.7.1). Thanks @MarcoRossignoli for your work. 🙏

**Reply by Eagle3386** [reply]
Sorry for the "resurrection", but just discovered Coverlet & I'm a bit confused: the release page states `5.x.x` as the most recent release while on NuGet, `3.x.x` is the latest.

Why's that, what's the difference & where's `4.x.x`? 😅

**Reply by MarcoRossignoli** [reply]
Version inside the repo and version on nuget are not the same, historically coverlet was shipped in 3 flavors with 3 different nuget version and the version inside the github release tab is the version of the core library shared by the flavors.
We consolidated the versioning for nuget to have less confusion. The version inside github is an internal one and as you can see the packages are versioned like in nuget.
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/7894084/202653898-08996465-5a78-4bdd-b0bb-aa805e988399.png">

**Reply by Eagle3386** [reply]
Thanks a lot for the info! _Very_ useful - maybe include that somewhere more "prominently", e. g. this repo's `README`?

Besides, just found #1122 - maybe now, almost 2 years later, you could reconsider this, i. e. consolidate everything as `6.x.x` to reflect the change? 🙂

**Reply by MarcoRossignoli** [reply]
Why not, we have to bump major to 4 for a breaking maybe we can go to 6 too cc: @daveMueller @petli

**Reply by petli** [reply]
Sure, good opportunity to get things in sync.

**Reply by daveMueller** [reply]
Yes I also don't mind. Sounds like a good plan.
I created an issue for that [https://github.com/coverlet-coverage/coverlet/issues/1413](https://github.com/coverlet-coverage/coverlet/issues/1413).
